central/xlsx/device.go

```diff
@@ -0,0 +1,174 @@
+//
+// Copyright (C) 2023 IOTech Ltd
+//
+
+package xlsx
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/common"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/dtos"
+
+	"github.com/xuri/excelize/v2"
+)
+
+// deviceXlsx stores the worksheets processed result and the converted Device DTOs
+type deviceXlsx struct {
+	fieldMappings  map[string]mappingField // fieldMappings defines all the device fields with default values defined in the xlsx
+	Devices        []*dtos.Device          `json:"devices"`
+	ValidateErrors []error                 `json:"validateErrors,omitempty"`
```

Comment:
the json tag is not necessary.
we should define DTO in the dto package to be converted to json, not in the convertor

Reply:
Should we add a `DeviceXlsxDTO` in the dto package and this deviceXlsx should probably be the model?

Reply:
This struct could be purely convertor.
How the client code to use this struct?  Do we have an API needs this DTO?
I thought we only need to return the error or success to the frontend.

Reply:
Done